AndroidBoundExample-support/src/main/java/solutions/alterego/androidbound/example/support/nestedrvs/viewmodel/MainNestedViewModel.java

```diff
@@ -0,0 +1,97 @@
+package solutions.alterego.androidbound.example.support.nestedrvs.viewmodel;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.experimental.Accessors;
+import solutions.alterego.androidbound.ViewModel;
+
+public class MainNestedViewModel extends ViewModel {
+
+    public List<Object> getDataItems() {
+        return this.mDataItems;
+    }
+
+    @Accessors(prefix = "m")
+    public static class RecyclerViewItem {
+
+        private String mName;
+
+        private String mImageUrl;
+
+        public RecyclerViewItem(String name, String imageUrl) {
+            mName = name;
+            mImageUrl = imageUrl;
+        }
+
+        public String getName() {
+            return this.mName;
+        }
+
+        public String getImageUrl() {
+            return this.mImageUrl;
+        }
+
+        public void setName(String mName) {
+            this.mName = mName;
+        }
+
+        public void setImageUrl(String mImageUrl) {
+            this.mImageUrl = mImageUrl;
+        }
+
+        public boolean equals(Object o) {
+            if (o == this) {
+                return true;
+            }
+            if (!(o instanceof RecyclerViewItem)) {
+                return false;
+            }
+            final RecyclerViewItem other = (RecyclerViewItem) o;
+            if (!other.canEqual((Object) this)) {
+                return false;
+            }
+            final Object this$mName = this.getName();
```

Comment:
why the $?

Reply:
hm, it was done automatically when delombokizing by the lombok plugin. :(

Reply:
do you want to change it ?